features/ssh/README.md

```diff
@@ -0,0 +1,12 @@
+# SSH
+
+## General
+Ssh (Secure Shell) is a program for logging into a remote machine and for executing commands on a remote machine. It provides secure encrypted communications between two untrusted hosts over an insecure network. X11 connections and arbitrary TCP/IP ports can also be forwarded over the secure channel. It can be used to provide applications with a secure communication channel.
```

Comment:
I wouldn't call the hosts untrusted. The network might be untrusted, though.

Reply:
Until you verified the host FP's it's untrusted. btw, this is taken from upstream man page.